### DISCUSSION TITLE
Send email immediately after SLA miss at dag level?

### DISCUSSION THREAD

**Discussion by match-gabeflores** [apache/airflow#34923] [Q&A] [answered]
We'd like to set some SLA misses for Dags. 

an example: dag with 10 tasks runs at 7am. If the dag doesn't finish by 8am, we'd like to send an email out but without stopping the job. 

SLAs are only at the task level?
https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/tasks.html#slas

relevant old question here: 
https://stackoverflow.com/questions/74642215/airflow-sla-on-dag-level

**Comment by Taragolis** [comment, marked as answer]
SLA mechanism is kind of "broken" for a long time, there is proposal for improvements/reimplementation, see [AIP-57](https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-57+Refactor+SLA+Feature)

**Reply by match-gabeflores** [reply]
Thanks, also following this issue
https://github.com/apache/airflow/issues/32816
